README.md

```diff
@@ -151,11 +151,52 @@ Exception handling while streaming events follows some simple rules
 
 You might want to ignore events that could not be mapped to your domain objects by Jackson, instead of having these events block all further processing. To achieve this you can override the `onMappingException` method of `Listener` and handle the `JsonMappingException` yourself.
 
-## Using another ClientHttpRequestFactory
+## Using another `ClientHttpRequestFactory` implementation
 
 This library is currently tested and used in production with `SimpleClientHttpRequestFactory` and `HttpComponentsClientHttpRequestFactory`.
 
-Please note that `HttpComponentsClientHttpRequestFactory` and also `SimpleClientHttpRequestFactory` since spring 4.3.x try to consume the remaining stream on closing and so might block during reconnection.
+The `HttpComponentsClientHttpRequestFactory` and `SimpleClientHttpRequestFactory` of spring try to consume
+the remaining stream on closing and so might block during reconnection. To avoid this there are two alternative
+implementations included in the artifacts `fahrschein-http-simple` and `fahrschein-http-apache`.
+
+```java
+
+final RequestConfig config = RequestConfig.custom().setSocketTimeout(readTimeout)
```

Comment:
Right, it's more showing the additional configuration possibilities of the apache impl. Also agree that there should be some guideline on when which impl should be used.